spec/requests/api/v1/trips/create_spec.rb

```diff
@@ -0,0 +1,47 @@
+RSpec.describe 'POST /api/v1/trips', type: :request do
+
+  describe 'Succesfully adds coords for destination' do
+    before do
+      post "/api/v1/trips",
+          params: { lat: "59.3293",
+            lng: "18.0685" }
+    end
+
+    it 'returns a 200 response status' do
+      expect(response).to have_http_status 200
+    end
+  end
+
+  describe 'Unsuccesfully when missing coords' do
+    before do
+      post "/api/v1/trips"
+    end
+
+
+    it 'returns a 200 response status' do
+      expect(response).to have_http_status 422
+    end
+
+    it 'returns error message' do
+      expect(response_json["error"]).to eq ["Lat can't be blank", "Lng can't be blank"]
+    end
+  end
+
+  describe 'Unsuccesfully when no city is nearby' do
+    before do
+      post "/api/v1/trips",
+      params: { lat: "9.3293",
+        lng: "18.0685" }
+    end
+
+
+    it 'returns a 200 response status' do
+      expect(response).to have_http_status 422
+    end
+
+    it 'returns error message' do
+      expect(response_json["error"]).to eq ["Destination can't be blank"]
+    end
+  end
+
+end
```

Comment:
Please get a good ruby formatter installed and spare me having to comment on mixing single and double quotes.